tasks/share_task_qc_atac.wdl

```diff
@@ -36,6 +34,7 @@ task qc_atac {
     String stats_log = '${default="share-seq" prefix}.atac.qc.stats.${genome_name}.log.txt'
     String hist_log = '${default="share-seq" prefix}.atac.qc.hist.${genome_name}.log.txt'
     String hist_log_pdf = '${default="share-seq" prefix}.atac.qc.hist.${genome_name}.log.pdf'
```

Comment:
This string is not required anymore

Reply:
See comment on commit 24e52e8; the name of the pdf is a required input to picard CollectInsertSizeMetrics

Reply:
Perfect,  then please add  the comment " the name of the pdf is a required input to picard CollectInsertSizeMetrics" so even in the future we will know why that string is required even if we are not using it and we don't want a pdf.

Reply:
My pleasure